build/.golangci.yml

```diff
@@ -104,10 +104,12 @@ linters-settings:
     # minimal occurrences count to trigger, 3 by default
     min-occurrences: 3
   depguard:
-    list-type: blacklist
-    include-go-root: false
-    packages:
-      - github.com/davecgh/go-spew/spew
+    rules:
+      main:
+        files:
+          - $all
```

Comment:
Why change this?

Reply:
Starting from v1.53, there have been breaking changes introduced in the configuration. For details https://github.com/golangci/golangci-lint/discussions/3727

Reply:
also met this when trying to upgrade it. thanks.